install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -882,6 +882,20 @@ data:
 {{- if .Values.hubble.redact.http }}
   # Enables redaction of the http URL query part in flows
   hubble-redact-http-urlquery: {{ .Values.hubble.redact.http.urlQuery | quote }}
+{{- if .Values.hubble.redact.http.headers }}
+  #  Enables redaction of http headers by specifying which headers will not be redacted
+{{- if .Values.hubble.redact.http.headers.allow }}
+  hubble-redact-http-headers-allow: {{- range .Values.hubble.redact.http.headers.allow }}
+    {{ . }}
+{{- end }}
+{{- end }}
+{{- if .Values.hubble.redact.http.headers.deny }}
+  # Enables redaction of http headers by specifying which headers will be redacted
+  hubble-redact-http-headers-deny: {{- range .Values.hubble.redact.http.headers.deny }}
+    {{ . }}
```

Comment:
Same here of course.

Reply:
Same as https://github.com/cilium/cilium/pull/26724#discussion_r1364486431